### ISSUE TITLE
Android widget

### ISSUE THREAD

**Issue by danielmosss** [Moustachauve/WLED-Native-Android#17]
https://github.com/Aircoookie/WLED/issues/3717#issuecomment-1925307909

Is your feature request related to a problem? Please describe.
No, this is not related to a problem.

Describe the solution you'd like
A 1x1 widget button where you can only press an on/off button. This will turn the ledstrip on or off. When you drag a widget on the screen for the first time you can configure it and have to select a ledstrip, color, brightness and effect.

Describe alternatives you've considered
I couldnt found a widget app that could use api and get this feature.

Thank you for your ideas for making WLED better!

**Comment by Moustachauve** [comment]
Thank you for your suggestion! Widgets are definitely planned and work for them is already ongoing :)

**Comment by brunokinast** [comment]
Just came here looking for this. Having some widgets to quickly control my LEDs would be really useful. Being able to select a preset from the widget would be cool too (the web UI from my WLED sometimes takes too long to load just to change a preset).

**Comment by danielmosss** [comment]
Yes that is also a small irritatition factor i came across, the app somethimes loads slow. 
I just installed the native android app and will try this one. I dont know it you already cache files? 
That would be a massive improvement but i dont now if that is possible. 
Would love to someday help thise wled community. Time to learn some new languages.

**Comment by zwamdurkel** [comment]
I would also love this. Just a simple on/off for a controller would be enough for me, but of course no harm is done by having more features for those who wish to have them.

**Comment by tu0mas** [comment]
Widgets still in the works? 👀

**Comment by Moustachauve** [comment]
Widgets are coming pretty soon! The first iteration will be very basic (just on/off), but more widget features are coming later.

**Comment by Moustachauve** [comment]
Should finally be available in the latest version, 7.0.0!
